build.sbt

```diff
@@ -191,3 +191,21 @@ ThisBuild / githubWorkflowBuildPostamble ++= Seq(
 ThisBuild / githubWorkflowUseSbtThinClient := false
 
 ThisBuild / githubWorkflowPublishTargetBranches := Seq()
+
+import ReleaseTransformations._
+
+releaseCrossBuild := true
```

Comment:
wondering if it worth to add `publishMavenStyle` to `true`.
cannot find much info, in [sbt-sonatype](https://github.com/xerial/sbt-sonatype#project-rootsonatypesbt) there was a comment:

```
// To sync with Maven central, you need to supply the following information:
publishMavenStyle := true
```

and I can see that [sbt-release itself sets it to true](https://github.com/sbt/sbt-release/blob/81966837213da6ecf7ca94f718404b83df33c3e9/build.sbt#L9)

?

Reply:
Its already done at https://github.com/zalando-nakadi/kanadi/blob/master/build.sbt#L133

Reply:
ah, sorry missed it